.github/ISSUE_TEMPLATE/ql--false-positive.md

```diff
@@ -0,0 +1,36 @@
+---
+name: False positive
+about: Tell us about an alert that shouldn't be reported
+title: False positive
+labels: false-positive
+assignees: ''
+
+---
+
+**Description of the false positive**
+
+<!-- Please explain briefly why you think it shouldn't be included. -->
+
+**Code samples or links to source code**
+
+<!--
+For open source code: file links with line numbers on GitHub, for example:
+https://github.com/github/codeql/blob/dc440aaee6695deb0d9676b87e06ea984e1b4ae5/javascript/ql/test/query-tests/Security/CWE-078/CommandInjection/exec-sh2.js#L10
+
+For closed source code: (redacted) code samples that illustrate the problem, for example:
+
+```
+function execSh(command, options) {
+    return cp.spawn(getShell(), ["-c", command], options) // <- command line injection
+};
+```
+-->
+
+**URL to the alert on GitHub CodeScanning (optional)**
+
+<!--
+1. Open the project on GitHub.com.
+2. Switch to the `Security` tab. For example, https://lgtm.com/projects/g/pallets/click/alerts/.
```

Comment:
I think that this URL should be updated to a code-scanning URL

Reply:
Good catch. I removed that URL. Such URLs are inaccessible for others, so a real example would just be a dead link for most viewers.